<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineedit_filename"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Template:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="combobox_template">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4K (3840x2160)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1080p (1920x1080)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>720p (1080x720)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="LineEdit" name="lineedit_width">
     <property name="text">
      <string>1920</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="LineEdit" name="lineedit_height">
     <property name="text">
      <string>1080</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineedit_path"/>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolbutton_browse">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="combobox_ext">
     <item>
      <property name="text">
       <string>.png</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>.jpg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>.ps</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>mfixgui.widgets.base</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
